server/server.js

```diff
@@ -1,7 +1,9 @@
 const express = require("express");
 const app = express();
+
 const path = require("path");
-const PORT = process.env.PORT || 3000;
+const port = process.env.PORT || 3000;
+app.listen(port, () => console.log(`Dragons spotted over on port ${port}`));
```

Comment:
Are you certain you want to move the app.listen to this line?  I believe any middleware/routes after this line won't be included in your app.  Make sure to test your api routes to make sure this works.

Reply:
There is an issue we are having with jest, and it really doesn't like app.listen for some reason. I would normally have it at the bottom of the page.